code/bmpman/bmpman.cpp

```diff
@@ -1506,8 +1539,8 @@ bitmap * bm_lock(int handle, ubyte bpp, ubyte flags, bool nodebug) {
 	// read the file data
 	if (gr_bm_lock(be->filename, handle, bitmapnum, bpp, flags, nodebug) == -1) {
 		// oops, this isn't good - reset and return NULL
-		bm_unlock(bitmapnum);
-		bm_unload(bitmapnum);
+		bm_unlock( handle );
```

Comment:
This looks like the correct way of using the handle. `bitmapnum` should not be passed to the `bm_*` functions.
Could you submit this as a separate PR as it's not really part of the EFF subdirectories patch?